index.html

```diff
@@ -0,0 +1,90 @@
+<!doctype html>
```

Comment:
Could you follow step 1 of the email instructions and update the PR accordingly?